src/Marten/Linq/Model/SelectManyQuery.cs

```diff
@@ -171,16 +171,35 @@ public void ConfigureCommand(IIncludeJoin[] joins, ISelector selector, CommandBu
 
         private string determineOrderClause(ChildDocument document)
         {
-            var orders = bodyClauses().OfType<OrderByClause>().SelectMany(x => x.Orderings).ToArray();
+            var orders = bodyClauses()
+                .SelectMany<IBodyClause, (Ordering Clause, bool CaseSensitive)>(x =>
```

Comment:
This block of code looks like a repetitive one as in `LinqQuery.cs` hence suggest to add a class to under `Util` and use it in both the places. Thoughts?

Reply:
I can certainly do that 👍 

One other thing came to my mind, should we also support:

```
session.Query<Target>()
                .Where(x => x.Id.In(ids))
                .OrderBy(x => x.String.ToLower())
                .ThenBy(x => x.AnotherString.ToLower())
```

This has the upside of allowing the chaning without the `IComparer` requirement that I explained in the description, but is less idiomatic in LINQ sense, both would produce the same SQL. (Of course Invariant versions and ToUpper also would be supported.

Reply:
there's now OrderingExtensions helper